ui/app/AppLayouts/Wallet/controls/TokenSelectorCompactButton.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Components 0.1
+import StatusQ.Components.private 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Control {
+    id: root
+
+    property bool selected
+    property bool forceHovered
```

Comment:
This at first looked like an antipattern; but if this were a `Button`, you could use the builtin `highlighted` property designed exactly for this purpose

Reply:
Hmm, and at a second look, what is this used for? :)

Reply:
I would prefer to keep the `Control` base here because `Button` introduces a massing api which is not needed here almost at all. Even `MouseArea` needs to stay inside. The `forceHovered` is not used indeed, I removed it.